### PR TITLE
fix: Color picker icon hover issue and menu font size revert issue

### PR DESCRIPTION
**Pull request by abhishek-bsf**
### Description

[Task](https://trello.com/c/d7f20phn/654-color-picker-improvement)

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Tested by hovering on Color Picker icon
- Tested by reverting menu font size

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests

### CODE REVIEW

File: inc/assets/css/ast-builder-customizer.css [hunk 1]
```diff
@@ -725,6 +725,11 @@
     border-color: #007cba;
 }
 
+
+.components-button.components-color-picker__saturation-pointer:not(:disabled):not([aria-disabled="true"]):not(.is-secondary):not(.is-primary):not(.is-tertiary):not(.is-link):hover{
```
Comment (rohitp-bsf) [inline inc/assets/css/ast-builder-customizer.css:729]: Selector seems too long, can we revamp this?

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]
LGTM!

**Comment by rohitp-bsf** [comment]
Also, Add a changelog.

File: inc/assets/css/ast-builder-customizer.css (hunk 1 above)
Comment (abhishek-bsf) [reply inc/assets/css/ast-builder-customizer.css]: Resolved! Removed this selector and added a condition to existing selector

**Pull request merged**